sw/device/benchmarks/coremark/README.md

```diff
@@ -1,13 +1,28 @@
-# Options
-There are two main options available for coremark SIM, ITERATIONS
+# Building Coremark
 
-## ITERATIONS
-Controls how many iteartions are run.  By default this value is 0 and will cause coremark to run >10s.
-For simulation and verilator, set to 1 for a reduced run
+To build coremark under meson:
 
-## For verilator
-make distclean; make SIM=1 ITERATIONS=1
-SIM=1 matces the verilator UART baudrates
+```sh
+cd "${REPO_TOP}"
+./meson_init.sh -f
```

Comment:
I guess in anticipation of https://github.com/lowRISC/opentitan/pull/1040 you can already skip the `-f` here.